dist/systemd/obs-delayedjob-queue-staging.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description = Open Build Service DelayedJob Queue: staging
+BindsTo = obs-api-support.target
+
+[Service]
+Environment = "RAILS_ENV=production"
+User = wwwrun
+Group = www
+WorkingDirectory = /srv/www/obs/api
+ExecStart = /usr/bin/bundle.ruby2.5 exec script/delayed_job.api.rb --queue=consistency_check start -i 1060
```

Comment:
@hennevogel, the queue should be `staging` and not `consistency_check`, right?

Reply:
true!